suite-native/module-onboarding/src/screens/AnalyticsConsentScreen.tsx

```diff
@@ -48,36 +111,91 @@ export const AnalyticsConsentScreen = () => {
+                            )}
                         />
                         <AnalyticsInfoRow
                             iconName="bugBeetle"
-                            title="What we collect"
-                            description="We gather info on application performance, visitor interaction, and potential technical issues to create a better user experience."
-                        />
-                        <AnalyticsInfoRow
-                            iconName="lock"
-                            title="We’re privacy junkies"
-                            description={<PrivacyDescription />}
+                            title={translate(
+                                'moduleOnboarding.analyticsConsentScreen.bulletPoints.dataCollection.title',
+                            )}
+                            description={translate(
+                                'moduleOnboarding.analyticsConsentScreen.bulletPoints.dataCollection.description',
+                            )}
                         />
                     </Stack>
-                </Card>
-                <Stack spacing="small" style={applyStyle(buttonsWrapperStyle)}>
-                    <Button data-testID="reject-userData" onPress={handleRedirect}>
-                        Not now
-                    </Button>
-                    <Button data-testID="allow-userData" onPress={handleAnalyticsConsent}>
-                        Allow anonymous data collection
-                    </Button>
-                </Stack>
-            </VStack>
-        </OnboardingScreen>
+                    <Box
+                        flexDirection="row"
+                        alignItems="center"
+                        flex={1}
+                        justifyContent="space-between"
+                        style={applyStyle(consentWrapperStyle)}
+                    >
+                        <Box flexDirection="row" style={{ gap: 8, paddingVertical: 4 }}>
+                            <Icon size="mediumLarge" name="info" color="iconPrimaryDefault" />
+                            <Text variant="hint" textAlign="center">
+                                <Translation id="moduleOnboarding.analyticsConsentScreen.helpSwitchTitle" />
+                            </Text>
+                        </Box>
+                        <Switch
+                            isChecked={isEnabled}
+                            onChange={enabled => {
+                                setIsEnabled(enabled);
+                            }}
+                        />
+                    </Box>
+                    <Text variant="hint" textAlign="center">
+                        <Translation
+                            id="moduleOnboarding.analyticsConsentScreen.learnMore.text"
+                            values={{
+                                securityLink: chunks => (
+                                    <Link
+                                        href={translate(
```

Comment:
I don't think we need to translate URLs, we don't have localized docs and there are no plans to do that.

Reply:
Done